src/DynamoCoreWpf/ViewModels/Core/HomeWorkspaceViewModel.cs

```diff
@@ -144,22 +145,22 @@ private void SetupFooterNotificationItems()
             footerItems[0] = new FooterNotificationItem()
             {
                 NotificationCount = 0,
-                NotificationImage = "/DynamoCoreWpf;component/UI/Images/error.png",
-                NotificationToolTip = "Click to cycle through nodes with errors.",
+                NotificationImage = Resources.FooterNotificationErrorImage,
+                NotificationToolTip = Resources.FooterNotificationErrorTooltip,
```

Comment:
These look unrelated to the bug, but good changes to be included. Did you find these while making the bug fixes?

Reply:
Yes I found it while fixing the tooltips direction